File: swagger/models/get_contracts_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .contract import Contract

class GetContractsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		contracts
		Contains information about each contract.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Contracts': ['contracts', Contract],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'contracts': 'Contracts',
		}

	pagination_response = None
	contracts = None

File: swagger/Models/GetContractsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetContractsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;
	/**
	 * Contains information about each contract.
	 * 
	 * @var Contract[]
	 */
	public $contracts;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class],
			'Contracts' => ['contracts', Contract::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => 'PaginationResponse',
			'contracts' => 'Contracts',
			];
	}
}
